### ISSUE TITLE
Wire up Email System

### ISSUE THREAD

**Issue by adamclmns** [PyArk/conference-website#15]
Email functionality needs to be setup. I don't know if we need to pay for something, or if we can use something in AWS.

Currently emails only show up in the logs.

**Comment by ericaroy** [comment]
I could probably work on this once it's decided which email to use if needed.

**Comment by lemonprogis** [comment]
Where is the domain name located?

**Comment by bheklilr** [comment]
@lemonprogis I don't think we have one yet.  We'd have to get something set up and no one has stepped up to make that decision.  Is that something you'd like to tackle?

**Comment by bheklilr** [comment]
It's all yours then, thanks =)
